library/Freckle/App.hs

```diff
@@ -259,6 +256,6 @@ instance
   where
   runSqlTx = runDB
 
-runAppT :: (MonadUnliftIO m, HasLogger app) => AppT app m a -> app -> m a
+runAppT :: MonadUnliftIO m => AppT app m a -> app -> m a
```

Comment:
I'm surprised this removed a constraint. I guess what happens now is this is perfectly usable without `HasLogger env` but when you go an actually use a `logInfo` (or whatever), it'll error for no instance _then_ since it can't solve `MonadLogger` without it?